src/index.html

```diff
@@ -8,6 +8,86 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="block_stars">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+      </div>
+
+      <div class="stars stars--2">
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+      </div>
+
+      <div class="stars stars--3">
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+      </div>
+
+      <div class="stars stars--4">
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star"></div>
+
+      </div>
+
+      <div class="stars stars--5">
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+        <div class="stars__star stars__star--active"></div>
+
+      </div>
+
+    </div>
```

Comment:
1. you can have empty lines only between stars blocks, it semms to be easier to read 
2. you can change `:nth-child` for stars to implement different stars color, you do not need to give every stars_star a modifier 😉 
read here:
 https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child
 
 example:
 `.stars--1 > .stars__star:nth-child(-n + 1)`

Reply:
Ok, I will change that. :)